data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			`	An Unfettered herald approaches and notices you looking around. "Incredible, is it not? To see hope on the face of buildings and on the face of people. I am here to say that it's time you learn how to communicate with us - with the True language of the Hai. Not only with us. You also need to be able to make demands to the birds, and there is only one appropriate language for it. We have learned that they too speak Hai, even if they disgrace our language by doing so."`
+			branch speakhai
+				not "language: Wanderer"
+			choice
+				`	"I already speak Hai. I have a translator, but I did't show it to you yet because I didn't know how you would react."`
+			`	"That is wonderful! Good initiative. But there still is something we would like you to do: scout all of the Wanderer systems for us, and bring back any information you may gather."`
+			choice
+				`	"I'll do it."`
+					accept
+				`	"No. I won't help you anymore."`
+					decline
+			label speakhai
+			choice
+				`	"How can I learn Hai?"`
+				`	"Is there some kind of translator?"`
+					goto translator
+			`	"Well to be frank, we don't have to ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`
+			label translator
+			`	"We think our Hai brethren may indeed have what we need, but we need a way to obtain it. In the meantime, you should go and meet the aliens up North and speak with them. They may not allow you to land on most of their planets. They may have grown distrusful of newcomers - probably because of our own actions. Go and see how pathethic these birds are, and then you should go to He Yi about that translator."`
+			branch neutral
+				has "reputation: Wanderer" >= 0
+			choice
+				`	"But I am at war with them already."`
+			`	"We know. Here is a message in Hai that you will transmit to them. It says that you are our ambassador, so do not open fire on them for now. You need to make the Hai see this new alien race as an ambassador between our races. The Hai would not believe anything if it came from us. We don't want to drag our brethren into a conflict that is ours. We are the warriors, not them. But they still need to see how weak the birds are."`
+			action
+				"reputation: Wanderer" = 0
+			label neutral
+			`	"Scout all of their systems for us, and bring back any information you may gather."`
+			choice
+				`	"I guess I'll try to land on their planets."`
+					accept
+				`	"I am not interested in being able to communicate with them."`
+			`	"Well if you want to work with us, you'll have to set that preferance aside. And don't forget: the translator is for communicating with us as well. Some among the most conversative of us despise speaking in your language. This will be a token of good faith on your part."`
```

Comment:
Did you mean`conservative`? Either works here so I couldn't tell.

Reply:
I didnt even realize I had written that

Reply:
```suggestion
			`	"Well if you want to work with us, you'll have to set that preference aside. And don't forget: the translator is for communicating with us as well. Some among the most conservative of us despise speaking in your language. This will be a token of good faith on your part."`
```
Here ya go!